.github/workflows/build.yml

```diff
@@ -27,11 +27,18 @@ jobs:
       image: quay.io/centos/centos:stream9
 
     steps:
+      - name: Enable additional repositories
+        run: |
+          dnf install -y dnf-plugin-config-manager
+
+          # CRB is required for meson
+          dnf config-manager -y --set-enabled crb
```

Comment:
This did not work for me. I had to do
```bash
sudo subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
```
as described in https://access.redhat.com/articles/4348511

Reply:
subscription-manager is only for RHEL, it shouldn't be used upstream